content/source/docs/enterprise/install/installer.html.md

```diff
@@ -12,6 +12,8 @@ installer to set up Terraform Enterprise on a customer-controlled machine.
 
 -> **Important:** Before you begin, consult the [Pre-Install Checklist](../before-installing/index.html) for prerequisites. You'll need to prepare data files and a Linux instance.
 
+-> **Important:** Be careful when cloning an already installed Terraform Enterprise host machine. You cannot have more than one Terraform Enterprise instance connect to the same database unless the installation is uses the [Active/Active architecture](active-active.html). When cloning a Terraform Enterprise host machine, you must make sure that the `/etc/sysconfig/replicated` or `/etc/default/replicated`, and `/etc/sysconfig/replicated-operator` or `/etc/default/replicated-operator` maintain the correct private IP address and the hostname of the Terraform Enterprise host machine before the restoration from the image. You should also restart the `replicated` and `replicated-operator` services when you update the configuration files.
+
```

Comment:
@hashitop I have another thought here. I agree with Judith that this is a lot to put in a callout box at the top of the page. Instead, I wonder if we can just make it a heading, like the others. I think that at least this way, users will be more likely to see it and the bullets will make it easier to parse. Thoughts? 

```suggestion
## Cloning a Host Machine

You cannot have more than one Terraform Enterprise instance connect to the same database unless the installation is uses the [Active/Active architecture](active-active.html). When cloning an existing Terraform Enterprise host machine:

-  Make sure that the `/etc/sysconfig/replicated` or `/etc/default/replicated`, and `/etc/sysconfig/replicated-operator` or `/etc/default/replicated-operator` maintain the correct private IP address and the hostname of the Terraform Enterprise host machine before the restoration from the image. 
- Restart the `replicated` and `replicated-operator` services when you update the configuration files.
```

Reply:
:thinking:  Having them that way, it becomes more like a suggestion/option which I don't believe that it is our recommendation or suggestion to clone TFE host machine, it is kind of something that can be done but with some extra cautions. It might be best to check with other teams on this as well but the content looks ok to me.

Reply:
@sudomateo Do you have thoughts on this/do you know anyone who would?

Reply:
From a technical perspective, cloning the instance can be a dangerous operation and only really acceptable in the following scenarios:
- When replacing an errant instance with a new instance that is as close of a replica as possible to the original instance. In this case both instances cannot be started at the same time due to database access issues.
- When creating an image template that can be used to recover the instance upon failure via automating scaling features. This type of cloning requires that the Terraform Enterprise application is not already installed among other cleanup items such as MAC address changes, etc.

With that being said, @hashitop what is the motivation or goal for this change? What information do we want the reader to be aware? Also, was there an issue or something that this information could have prevented?

Reply:
My initial thought would be just to provide a note for extra caution for whoever wants to implement this.
I don't think editing those files is a common thing that we would normally do. If we would not recommend this method at all then perhaps we would rather explicitly state that we don't support it. 

And as you mentioned that it is a dangerous operation, we could provide a separate set of instructions for backup and restore data into another machine or a different method for people to clone TFE instance? I do see that we have a page for backup/restore API but we also have other methods in TFE as well and for mounted disk, perhaps the CLI maybe faster and more reliable? When thinking about this, I am actually not sure why some people did this, perhaps we did not provide sufficient details on how to use the database backup/restore command on mounted disk mode? Maybe we need to rethink again but the initial thought was only to provide a set of cautions and a way to avoid the error providing that we did not explicitly state anything to this method currently.

Reply:
Thank you for the context, both!  If some non-trivial number of people are doing this/will want to do it, we should address it and then give our recommendations. Maybe we should add what @sudomateo wrote at the top of this new section along with an explicit warning that we don't recommend this? 

But I'm open to whatever y'all think is best here, as you have much more experience with the customers than I do. thoughts?